Thesis.tex

```diff
@@ -162,6 +162,22 @@ \section{Business Models}
+
+The idea was later formalized into the four freedoms that make software free software as postulated by the Free Software Foundation: "When you speak about Free Software, you speak about freedom. And more precisely, about the four freedoms to use, study, share and improve the software." (citation needed https://fsfe.org/freesoftware/basics/4freedoms.en.html).
+
+This Idea is in stark contrast to proprietary (closed source) software. One might conclude that OSS thus is a hobby, not a business, as you can't sell the same software anyone can compile and download. This, however, is not true and was already anticipated at the time of the GNU manifesto, which states: "If people would rather pay for GNU plus service than get GNU free without service, a company to provide just service to people who have obtained GNU free ought to be profitable." (citation GNU Manifesto https://www.gnu.org/gnu/manifesto.en.html). The business model around OSS can thus be characterized as selling services to customers. There are several services that can be provided for a charge, e.g.:
+\begin{itemize}
+	\item Support
+	\item Enhancements
+	\item Customisations
+	\item Education 
+\end{itemize}
+
+The question is how one can credibly offer good services and be competitive when everyone else could do the same? It comes down to reputation. If someone has contributed to the programming of a software, this person (legal or natural) gets recognised for that effort (see also https://en.wikipedia.org/wiki/Open-source\_software\_movement). This recognition of the effort and the knowledge of a system can be used to bring leverage to the marketing efforts for selling services around the software. Thus, it is not surprising that big corporations have stepped in to contribute to OSS as the example of Linux, a free operating system, clearly shows. In their annual announcement of the Linux Development Report 2015, the Linux Foundation mentions "The Top 10 organizations sponsoring Linux kernel development since the last report include Intel, Red Hat, Linaro, Samsung, IBM, SUSE, Texas Instruments, Vision Engraving Systems, Google and Renesas." (citation https://www.linuxfoundation.org/news-media/announcements/2015/02/linux-foundation-releases-linux-development-report). While Suse and Redhat are primarily Linux distributors, others use Linux in their products and services. The example of Google, IBM and Samsung illustrate that the interest in OSS is not only by SMEs, but also big, worldwide corporations.
```

Comment:
[...] the interest in OSS not only comes from _small and medium sized enterprises_, but also big, _global_ corporations.
I know SME is a common abbreviation, but I would use it when I used SMEs multiple times in a text. And even then, I would define it first.